components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -172,13 +172,10 @@ function InputBox(props: InputBoxProps) {
           {props.context.isUploadingFiles && (
             <AttachmentSpinnerItem title={getLocale(S.AI_CHAT_UPLOADING_FILE_LABEL)} />
           )}
-          {showImageAttachments && props.context.pendingMessageImages?.map((img, i) => (
-            <AttachmentImageItem
-              key={img.filename}
-              uploadedImage={img}
-              remove={() => props.context.removeImage(i)}
-            />
-          ))}
+          <AttachmentUploadItem
```

Comment:
nit: maybe this should be plural, as we're rendering multiple now

Reply:
renamed in 53509362ade970cc99956e5006a8665150ca4136